manifests/personal.pp

```diff
@@ -78,9 +78,7 @@
   # If any casks/osx_apps are specified, declare them as brewcask packages
   if count($_casks) > 0 { include brewcask }
   ensure_resource('package', $_casks, {
-    'provider'        => 'brewcask',
-    'install_options' => ['--appdir=/Applications',
-                          "--binarydir=${boxen::config::homebrewdir}/bin"],
+    'provider'        => 'brewcask'
```

Comment:
Can we remove the extra whitespace in between here? Now that the other options are gone, this shouldn't be required.